package.json

```diff
@@ -53,6 +53,7 @@
     "@wordpress/edit-post": "^7.28.1",
     "classnames": "^2.2.6",
     "eslint": "8.56.0",
+    "hammerjs": "^2.0.8",
```

Comment:
https://github.com/greenpeace/planet4-master-theme/pull/2618/files#r2077429381

Reply:
Hopefully this will also solve [these Sentry errors ](https://greenpeace-international.sentry.io/issues/6897044721/?query=is%3Aunresolved%20hammer&referrer=issue-stream)🤞